install/kubernetes/helm/istio/charts/kiali/values.yaml

```diff
@@ -5,7 +5,7 @@ enabled: false # Note that if using the demo or demo-auth yaml when installing v
 replicaCount: 1
 hub: quay.io/kiali
 image: kiali
-tag: v1.9
+tag: v1.15
```

Comment:
does kiali 1.15 work well with istio 1.4?

Reply:
We already made this change for istioctl install but missed it in helm